vue-frontend/src/components/contact/NewContactForm.vue

```diff
@@ -0,0 +1,576 @@
+                v-model="reference"
+                :disabled="disableInput"
+                placeholder=" "
+                @input="
+                  showReferenceValidationError = reference.trim().length === 0
+                "
+              />
+              <label
+                for="reference"
+                class="tw-absolute tw-top-7 tw-left-0 tw-z-[2] tw-text-black-core/[0.6] tw-font-inter-regular tw-text-[1rem] md:tw-text-[1.375rem] tw-leading-[1.21rem] md:tw-leading-[2.0625rem] tw-transform tw--translate-y-4 tw-origin-[0] tw-scale-75 tw-duration-300 peer-focus:tw-text-black-core/[0.6] peer-placeholder-shown:tw-scale-100 peer-placeholder-shown:tw-translate-y-0 peer-focus:tw-scale-75 peer-focus:tw--translate-y-4"
+                >How did you find us?</label
+              >
+              <span
+                v-if="showReferenceValidationError"
+                class="error tw-text-red-600"
+                >This field is required</span
+              >
+            </div>
+            <div class="tw-relative tw-pt-6">
+              <div ref="invest-list" class="tw-flex">
+                <button
+                  class="tw-flex tw-mt-4 tw-items-center tw-justify-between tw-mx-0 tw-w-full tw-px-0 tw-border-b tw-border-slate-400 tw-bg-none tw-font-inter-regular tw-text-[1rem] md:tw-text-[1.375rem] tw-leading-[1.21rem] md:tw-leading-[2.0625rem] tw-whitespace-nowrap tw-transition tw-duration-150 tw-ease-in-out focus:tw-outline-0 active:tw-outline-0 focus:tw-shadow-none active:tw-shadow-none focus:tw-ring-0 active:tw-ring-0 focus:tw-bg-transparent active:tw-bg-transparent active:tw-text-black-core/[0.6]"
+                  type="button"
+                  id="invest"
+                  name="invest"
+                  required
+                  :disabled="disableInput"
+                  @click="toggleList"
+                >
+                  <label
+                    for="invest"
+                    class="tw-absolute tw-top-7 md:tw-top-8 tw-left-0 tw-text-black-core/[0.6] tw-transform tw-duration-300 placeholder:tw-!text-black-core/[0.6]"
+                    :class="[
+                      floatable
+                        ? 'tw--translate-y-4 xl:tw--translate-y-6  tw-origin-[0] tw-scale-75  '
+                        : 'tw-translate-y-0 tw-scale-100',
+                      floatable && showList
+                        ? 'tw-text-white'
+                        : 'tw-text-white/[.6]',
+                    ]"
+                    @click="floatable = !floatable"
+                    >I'll invest</label
+                  >
+                  <span class="tw-cursor-text">{{ invest }}</span>
+                  <font-awesome-icon
+                    class="fab tw-my-auto tw-w-[15px] md:tw-w-[25px] tw-h-[15px] md:tw-h-[25px] tw-text-black-core/[0.6]"
+                    :icon="showList ? faCaretUp : faCaretDown"
+                  />
+                </button>
+                <ul
```

Comment:
Background and font color and size should be matched with other fields. Also, it should be below or aligned with the bottom line of the input field. Currently its above the line, not looking perfact